docs/tutorials/autogluon-cloud.md

```diff
@@ -144,17 +144,24 @@ A general guideline is to use batch inference if you need to get predictions les
 To perform batch inference:
 
 ```{.python}
-cloud_predictor.predict(
+result = cloud_predictor.predict(
     'test.csv',  # can be a DataFrame as well and the results will be stored in s3 bucket
     instance_type="ml.m5.2xlarge",  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
-    wait=True  # Set this to False to make it unblocking call
-)
-cloud_predictor.download_predict_results(
-    job_name=None  # download the most recent finished batch inference results to your local machine. Specify the job name to download a specific batch inference job's results.
-    save_path="PATH"  # If not specified, CloudPredictor will create one.
+    wait=True,  # Set this to False to make it unblocking call
+    download=True  # Batch transform results are stored into S3. Set this to False if you don't want to download the results.
```

Comment:
If True, returns a Pandas Series object of predictions.
If False, returns nothing. You will have to download results separately via `cloud_predictor.download_predict_results`